content/zh-cn/docs/concepts/configuration/secret.md

```diff
@@ -645,7 +645,7 @@ Kubernetes 所提供的内置类型 `kubernetes.io/ssh-auth` 用来存放 SSH 
 <!--
 The SSH authentication Secret type is provided only for convenience.
 You can create an `Opaque` type for credentials used for SSH authentication.
-However, using the defined and public Secret type (`kubernetes.io/ssh-auth`) helps other
+However, using the defined and public Secret type (`kubernetes.io/tls`) helps other
```

Comment:
Line 655 in zh para shall be changed too.

Reply:
Done, thanks